app/controllers/admin/salesforce_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Admin
+  class SalesforceController < BaseController
+    layout 'admin'
+
+    def show
+    end
+
+    def callback
+      user = SalesforceUser.save_from_omniauth!(env["omniauth.auth"])
+      SalesforceUser.all.reject{|uu| uu.id == user.id}.each(&:destroy)
```

Comment:
Whoa... This makes me wonder if SalesforceUser should even be an ActiveRecord thing (couldn't it just be something handled by rails-settings-cached?)